joomla/joomla-cms
# [6.0] Article edit tabs and fields

Pull Request for Issue #44118 .

### Summary of Changes
This PR re-arranges fields and tabs in frontend edit for articles. 
It moves all options which are in the right sidbar of articles to a new tab "options"
It writes all fields which are in the publishing tab of the article into the publishing tab.
This removes the language tab and the metadata tab.



### Testing Instructions
Compare the edit form for an article before and after the PR. Make sure that different user groups witrh different permissions have the same fields before and after the pr. 
Swith on and off the show_publishing_data in articles options. on: publishing tab exists, off: no publishing data. 


### Actual result BEFORE applying this Pull Request
see #44118


### Expected result AFTER applying this Pull Request
![grafik](https://github.com/user-attachments/assets/cb1e5f68-8c29-472c-9a0c-1a510ffcfb59)
![grafik](https://github.com/user-attachments/assets/ad78b653-d525-4dfc-8e4f-05fa2ced6bfa)

**REMARK**
It would be correct to use a new param for switching on and off the publishing tab. But this would be a b/c break.


### Link to documentations
Please select:
